src/Quartz/CronExpression.cs

```diff
@@ -269,84 +278,127 @@ public class CronExpression : ICloneable, IDeserializationCallback
         private static readonly Dictionary<string, int> monthMap = new Dictionary<string, int>(20);
         private static readonly Dictionary<string, int> dayMap = new Dictionary<string, int>(60);
 
+        [DataMember]
         private readonly string cronExpressionString;
 
         private TimeZoneInfo timeZone;
 
+        // Serializing TimeZones is tricky in .NET Core. This helper will ensure that we get the same timezone on a given platform,
+        // but there's not yet a good method of serializing/deserializing timezones cross-platform since Windows timezone IDs don't
+        // match IANA tz IDs (https://en.wikipedia.org/wiki/List_of_tz_database_time_zones). This feature is coming, but depending
+        // on timelines, it may be worth doign the mapping here.
+        // More info: https://github.com/dotnet/corefx/issues/7757
+        [DataMember]
+        private string timeZoneInfoId
+        {
+            get { return timeZone?.Id; }
+            set { timeZone = (value == null ? null : TimeZoneInfo.FindSystemTimeZoneById(value)); }
```

Comment:
Ah, I didn't see that. Cool. I'll call that method instead of TZI.FindSystemTimeZoneById directly and if we need better mapping, we can make changes in TimeZoneUtil.